techniques/css/C38.html

```diff
@@ -178,34 +170,27 @@ <h3>Example 1: Fitting labels, inputs and flexbox layout with HTML and CSS.</h3>
   &lt;/div&gt;
 &lt;/div&gt;
 </code>
-</pre><p class="working-examples">Working example: <a href="../../working-examples/css-fit-labels-inputs/">Using Adjustable Labels and Inputs for Reflow</a></p>
+</pre><p class="working-examples">動作例: <a href="../../working-examples/css-fit-labels-inputs/">Using Adjustable Labels and Inputs for Reflow</a></p>
             </section>
          </section>
          <section id="tests">
-            <h2>Tests</h2>
+            <h2>検証</h2>
             <section class="test-procedure">
-               <h3>Procedure</h3>
+               <h3>手順</h3>
                <ol>
-                  
-                  <li>Display the web page in a user agent capable of 400% zoom and set the viewport dimensions
-                     (in CSS pixels) to 1280 wide and 1024 high.
-                  </li>
-                  
-                  <li>Zoom in by 400%.</li>
-                  
-                  <li>For vertically scrolling content, all labels and inputs fit in their available space
-                     without horizontal scrolling.
-                  </li>
-                  
+                  <li>コンテンツを 400% ズームが可能なユーザーエージェント上で、ビューポートサイズを 1280x1024 CSSピクセルに設定してウェブページを表示する。</li>
```

Comment:
```suggestion
                  <li>コンテンツを 400% ズームが可能なユーザエージェント上で、ビューポートサイズを 1280x1024 CSSピクセルに設定してウェブページを表示する。</li>
```

面倒なことこの上ないルールですが、長音を省きます。